src/odemis/driver/static.py

```diff
@@ -103,9 +104,10 @@ def __init__(self, name, role, mag, mag_choices=None, na=0.95, ri=1,
             if (not isinstance(pole_pos, collections.Iterable) or
                 len(pole_pos) != 2 or any(not 0 < v < 1e6 for v in pole_pos)):
                 raise ValueError("pole_pos must be 2 positive values, got %s" % pole_pos)
-            self.polePosition = model.ResolutionVA(tuple(pole_pos),
-                                                   rng=((0, 0), (1e6, 1e6)),
-                                                   unit="px")
+            self.polePosition = model.TupleContinuous(tuple(pole_pos),
```

Comment:
Do we want to add the range to the docstring? Since this input is directly used from the __ init__ maybe it is usefull information when looking at the signature of the constructor. Or is this something we choose to not add to the docstring?

Reply:
Which docstring are you talking about? The one of OpticalLens? In the docstring, we've already indicated that it should be > 0, so only the max is needed. (actually, it's >=0, I'll fix that). However, here the reason I didn't specify the max is because I picked "1e6" arbitrarily as just a big number which no CCD should have more pixels than that.